platform/android/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
+## 6.0.2
```

Comment:
I targeted this PR at the release-boba branch, with the intention of getting these translations into iOS map SDK v4.0.1. However, I’m unsure which release and which branch would be appropriate for the Android map SDK.